db/seeds.rb

```diff
@@ -184,4 +183,96 @@
 texture_exploring.category = cognitive
 texture_exploring.save!
 
+puts "Creating physical activities"
+Activity.create(
```

Comment:
@goh0222 just to check, i noticed your database layout is different from yaamanni's one. 
what is the better way to save database data? cuz I will want to add some tomorrow

Reply:
No hard rules on this @Rainesisu